### PR TITLE
Fixed MIUI devices bug showing the app in blank (finally)

### LINKED ISSUE #173: [Bug Report] Blank Screen
### Describe the bug

Is that a bug in Navigation in Compose library ?
I also face the same bug in my app when navigating to other screens 

### To Reproduce

1. Repeatedly navigating to other screens will throw a blank screen.

### Error reports & Screenshots

https://user-images.githubusercontent.com/68665948/187888648-9ebb5f36-96a6-4d7b-b56a-9c42961299be.mp4

### Device info

1.0.5 (API 29)
[arm64-v8a, armeabi-v7a, armeabi]

### Additional context

_No response_

### LINKED ISSUE #231: Gray screen
### Describe the bug

When I rotate the screen 2 times, I get a gray screen. It can be fixed by pressing "back" 2 times and exiting the app. After restarting the app, everything works

In the menu where you enter the link this is fixed, but it also shows up for a second. Elsewhere it is not fixed

### To Reproduce

Rotate the screen twice in "settings"

### Error reports & Screenshots

_No response_

### Device info

App version: 1.3.1 (10310)
Device information: Android 12 (API 31)
[arm64-v8a, armeabi-v7a, armeabi]
Screen resolution: 2263 x 1078
MIUI 13.0.5

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by BobbyESP**
As the title says

Closes #231 
Closes #173

### CODE REVIEW

**Comment by BobbyESP** [comment]
https://user-images.githubusercontent.com/60316747/197338679-7895b122-fdd8-4c75-bfa1-4a662f05f6c6.mp4

**Comment by BobbyESP** [comment]
That visual bugs in the video is because video compression.

File: app/src/main/java/com/junkfood/seal/ui/theme/Theme.kt [hunk 1]
```diff
@@ -60,6 +55,16 @@ fun SealTheme(
         )
         else this
     }
+    val view = LocalView.current
+    if(!view.isInEditMode){
+        val currentWindow = (view.context as? Activity)?.window
+        SideEffect {
+            (view.context as Activity).window.statusBarColor = android.graphics.Color.TRANSPARENT
+            (view.context as Activity).window.navigationBarColor = android.graphics.Color.TRANSPARENT
+            WindowCompat.getInsetsController(currentWindow!!, view).isAppearanceLightStatusBars =
```
Comment (JunkFood02) [inline app/src/main/java/com/junkfood/seal/ui/theme/Theme.kt:64]: The fix that really works for the bug seems to be here

**Review by JunkFood02 — APPROVED** [review]

**Comment by JunkFood02** [comment]
LGTM👍

File: app/src/main/java/com/junkfood/seal/ui/theme/Theme.kt (hunk 1 above)
Comment (BobbyESP) [reply app/src/main/java/com/junkfood/seal/ui/theme/Theme.kt:64]: Yes. By some reason, rememberSystemUiController.run was causing the UI not getting updated.

**Pull request merged**